--- src/local/dotforward.c.old	2012-02-12 17:35:34.000000000 +0100
+++ src/local/dotforward.c	2012-02-12 17:36:23.000000000 +0100
@@ -231,12 +231,12 @@
 	    state.msg_attr.exp_from = state.msg_attr.local;
 	    if (S_ISREG(st.st_mode) == 0) {
 		msg_warn("file %s is not a regular file", STR(path));
-	    } else if (st.st_uid != 0 && st.st_uid != usr_attr.uid) {
+	    } /*else if (st.st_uid != 0 && st.st_uid != usr_attr.uid) {
 		msg_warn("file %s has bad owner uid %ld",
 			 STR(path), (long) st.st_uid);
 	    } else if (st.st_mode & 002) {
 		msg_warn("file %s is world writable", STR(path));
-	    } else if ((fd = open_as(STR(path), O_RDONLY, 0, usr_attr.uid, usr_attr.gid)) < 0) {
+	    } */else if ((fd = open_as(STR(path), O_RDONLY, 0, usr_attr.uid, usr_attr.gid)) < 0) {
 		msg_warn("cannot open file %s: %m", STR(path));
 	    } else {
 
